test/cpp/src/TestClient.cpp

```diff
@@ -488,7 +488,7 @@ int main(int argc, char** argv) {
       testClient.testBinary(bin_result, string(bin_data, 256));
       if (bin_result.size() != 256) {
         printf("}\n*** FAILED ***\n");
-        printf("invalid length: %lu\n", bin_result.size());
+        printf("invalid length: %zu\n", bin_result.size());
```

Comment:
Why constructing such complicated boost format line, when simple `cout <<` is enough? :) simpler code == better code

Also - it is true - long term development should remove boost, especially from compiler, to allow maximum ease of thrift portability for 'non-C++' users